executor_manager/services/sandbox/manager.py

```
@@ -0,0 +1,961 @@
+            logger.info(f"[SandboxManager] Trying direct lookup with task_id={task_id}")
+            execution = self._repository.load_execution(task_id, subtask_id)
+            if execution:
+                logger.info(f"[SandboxManager] Found execution by task_id={task_id}")
+                return execution
+            logger.info(
+                f"[SandboxManager] No execution found for task_id={task_id}, subtask_id={subtask_id}"
+            )
+        except ValueError:
+            # sandbox_id is not numeric, might be e2b_sandbox_id
+            logger.info(
+                f"[SandboxManager] sandbox_id is not numeric, trying e2b_sandbox_id lookup"
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)